packages/user-service/src/cron.ts

```diff
@@ -0,0 +1,46 @@
+import * as _ from 'lodash';
+import * as async from 'async';
```

Comment:
Also, can you avoid using the external async library? The inbuilt Promise is sufficient to handle the use-case

Reply:
removed the usage